.travis.yml

```diff
@@ -34,6 +34,8 @@ env:
     - E2E_OUTPUT=base64
     # Enable end-to-end tests
     - E2E=no
+    # Force pipenv to create a new virtualenv to bypass travis virtualenv.
+    - PIPENV_IGNORE_VIRTUALENVS=1
```

Comment:
If you add this, perhaps, you can remove the `|| true` from https://github.com/inveniosoftware/cookiecutter-invenio-instance/blob/master/run-tests.sh#L22, it was there to ignore the error `pipenv` raises when trying to remove a virtualenv it didn't create.